episodes/image-display.md

```diff
@@ -0,0 +1,358 @@
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Image array and display
+
+Last episode we saw that images are arrays of numbers with specific dimensions 
+and data type. Napari reads these numbers (pixel values) and converts them into 
+colours on our display, allowing us to view the image. Exactly how this 
+conversion is done can vary greatly, and is the topic of this episode.
+
+For example, take the image array shown below. Depending on the display 
+settings, it can look very different inside Napari:
+
+![](fig/same-array-diff-display.png
+){alt="Diagram showing an image array (top) with three different colormap 
+options (bottom)" width='70%'}
+
+Equally, image arrays with different pixel values can look the same in Napari 
+depending on the display settings:
+
+![](fig/diff-array-same-display.png
+){alt="Diagram showing two image arrays - 8-bit vs 16-bit (top) with the same 
+display (bottom)" width='50%'}
+
+In summary, we can't rely on appearance alone to understand the underlying pixel 
+values. Display settings (like the colormap, brightness and contrast - as we 
+will see below) have a big impact on how the final image looks.
+
+## Napari plugins
+
+How can we quickly assess the pixel values in an image? We could hover over 
+pixels in Napari, or print the array into Napari's console (as we saw last 
+episode), but these are hard to interpret at a glance. A much better option is 
+to use an image histogram.
+
+To do this, we will have to install a new plugin for Napari. Remember from 
+episode 1 that plugins add new features to a piece of software. Napari has 
+hundreds of plugins available on the [napari hub](https://www.napari-hub.org/) 
+website.
+
+Let's start by going to the napari hub and searching for 'matplotlib':
+
+![](fig/napari-hub.png
+){alt="Screenshot of searching 'matplotlib' on napari hub"}
+
+You should see 'napari Matplotlib' appear in the list (if not, try scrolling 
+further down the page). If we click on `napari matplotlib` this opens a summary 
+of the plugin with links to the documentation. There's also a useful 'Activity' 
+tab that summarises the number of installs and maintenance history of the plugin.
+
+Now that we've found the plugin we want to use, let's go ahead and install it 
+in Napari. Note that some plugins have special requirements for installation, 
+so it's always worth checking their napari hub page for any extra instructions. 
+In the top menu bar of Napari select:  
+`Plugins > Install/Uninstall Plugins...`
+
+![](fig/plugin-installation.png
+){alt="Screenshot of plugin installation window in Napari"}
+
+This should open a window summarising all installed plugins (at the top) and all 
+available plugins to install (at the bottom). If we search for 'matplotlib' in 
+the top searchbar, then 'napari-matplotlib' will appear under 'Available 
+Plugins'. Press the blue install button and wait for it to finish. You'll then 
+need to close and re-open Napari.
```

Comment:
```suggestion
Plugins'. Press the blue install button and wait for it to finish. **You'll then 
need to close and re-open Napari.**
```
Need to make this really clear to the user...bold, separate line of text something.